MosipNexus/docs/generate_docs.py

```
@@ -0,0 +1,1601 @@
+
+def build_developer_guide():
+    doc = Document()
+    doc.styles["Normal"].font.name = "Calibri"
+    doc.styles["Normal"].font.size = Pt(11)
+
+    # Cover
+    doc.add_paragraph()
+    t = doc.add_paragraph()
+    t.alignment = WD_ALIGN_PARAGRAPH.CENTER
+    r = t.add_run("MOSIP Nexus")
+    r.font.size = Pt(32); r.font.bold = True; r.font.color.rgb = BLUE
```

Ruff (0.15.14)

[error] Multiple statements on one line (semicolon)

(E702)